Extensions/forms.md

```diff
@@ -95,6 +95,7 @@ Used for grouping related `elements` into rows.
 - select
 - radioGroup
 - checkboxes
+- multiselect
```

Comment:
We don't have a multi select form component. Building a multi select filter component won't make it available for use in the forms.